dcos/dcos-ui
# DCOS-55631 Quota Overview

Closes https://jira.mesosphere.com/browse/DCOS-55631 .

## Testing

1. Login to DC/OS UI
2. Update your `Config.dev.ts`
```
uiConfiguration: {
...
  features: {
    quota: true
  }
},
  useFixtures: true,
  ...
```
3. Go to services and open the Quota tab.
4. Verify that the table is shown correctly and follows the design - exact columns, percentages, tooltip when hovered, "no limit" state, etc.
5. Verify that the table is sortable.
6. Verify that the added tests make sense.

## Trade-offs
- Actions are not in scope here, so actions column won't be rendered now.

- The table has a weird transition when sorting.
![Peek 2019-07-02 11-12](https://user-images.githubusercontent.com/40791275/60495880-55de6880-9cba-11e9-9682-2e8597bb17e1.gif)

## Dependencies
None.

## Screenshots
![Screenshot from 2019-07-08 17-51-58](https://user-images.githubusercontent.com/40791275/60819920-18da1080-a1a9-11e9-805d-af6315a7da3c.png)
